tests/pages/api/chooseStages.test.ts

```diff
@@ -45,7 +45,7 @@ describe('chooseStages', () => {
 
     it.only('should set the fare stages cookie according to the specified number of fare stages', () => {
```

Comment:
Just spotted that this it.only has been committed in a previous merge. Should we add lint rule(s) for skipping tests on commit?